portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -42,6 +42,14 @@
 @RunWith(JUnit4.class)
 public final class DataServletTest {
   static final String COMMENT_TABLE_NAME = "Comment";
+  static final String COMMENT_COLUMN_NAME = "text";
+  static final String TIMESTAMP_COLUMN_NAME = "submit_time";
+  HttpServletRequest postRequest = mock(HttpServletRequest.class);
+  HttpServletResponse postResponse = mock(HttpServletResponse.class);
+  HttpServletRequest getRequest = mock(HttpServletRequest.class);
+  HttpServletResponse getResponse = mock(HttpServletResponse.class);
```

Comment:
I think these can be consolidated into a single request and a single response. A test method won't exercise both since we're using the datastore to populate things...

Reply:
I have a couple of tests that are doing both Posts and Gets...should I remove those tests?

Reply:
done!